packages/jaeger-ui/src/components/App/TopNav.tsx

```diff
@@ -96,9 +96,14 @@ function getItem(item: ConfigMenuItem) {
 function CustomNavDropdown({ label, items }: ConfigMenuGroup) {
   const menuItems = <Menu>{items.map(getItem)}</Menu>;
   return (
-    <Dropdown overlay={menuItems} placement="bottomRight">
-      <a>
-        {label} <DownOutlined />
+    <Dropdown
+      trigger={['click']}
+      overlay={menuItems}
+      overlayStyle={{ paddingRight: '20px' }}
```

Comment:
This gives the overlay of dropdown (The panel that opens when someone clicks the About Jaeger Menu item), a padding. It was changed after the upgrade to v4.16.0